src/game/default/g_types.h

```diff
@@ -698,6 +698,8 @@ typedef struct {
 	char skin[MAX_QPATH];
 	g_hand_t hand;
 
+	vec_t handicap; // handicap inverse percentage from 0 to 100
+  
```

Comment:
I never used Q3's handicap feature, so I'm not familiar with all of it's functionality. But from what @maci0 said above, apparently your handicap would also make your weapons that much less effective. I do not have a strong opinion on whether that is needed or not.